mage_ai/frontend/components/PipelineDetail/UploadPipeline/index.tsx

```diff
@@ -0,0 +1,175 @@
+import router from 'next/router';
+import { useMemo, useState } from 'react';
+
+import ApiErrorType from '@interfaces/ApiErrorType';
+import Button from '@oracle/elements/Button';
+import FileUploader from '@components/FileUploader';
+import FileType from '@interfaces/FileType';
+import FlexContainer from '@oracle/components/FlexContainer';
+import Panel from '@oracle/components/Panel';
+import PopupMenu from '@oracle/components/PopupMenu';
+import ProgressBar from '@oracle/components/ProgressBar';   
+import Spacing from '@oracle/elements/Spacing';
+import Table from '@components/shared/Table';
+import Text from '@oracle/elements/Text';
+import ToggleSwitch from '@oracle/elements/Inputs/ToggleSwitch';
+import { DropZoneStyle, TableStyle } from './index.style';
```

Comment:
@PopaRares Can you just import from `mage_ai/frontend/components/FileBrowser/UploadFiles/index.style.tsx`?